src/CameraGstreamer.cpp

```diff
@@ -13,6 +13,8 @@ gboolean CameraGstreamer::busCallback( GstBus *bus, GstMessage *msg, gpointer da
     GError *err = NULL;
     gchar *debug = NULL;
 
+    std::cout << "Bus callback called with message of type: " << GST_MESSAGE_TYPE_NAME(msg) << std::endl;
```

Comment:
That line might produce a lot of prints

Reply:
I wish it would :(
I don't see that this is callback is **ever** called.